### ISSUE TITLE
also publish sdist

### ISSUE THREAD

**Issue by tacaswell** [scipy/oldest-supported-numpy#7]
Because oldest-supported-numpy only publishes a wheel to pypi you can not install packages that use the new build system with `--no-binary :all:`.  

see https://github.com/liberfa/pyerfa/issues/51 for an example

**Comment by rgommers** [comment]
Thanks @tacaswell, that is indeed a good reason to ship an sdist.

**Comment by astrofrog** [comment]
Interestingly PyPI doesn't let me upload an sdist:

```
HTTPError: 400 Bad Request from https://upload.pypi.org/legacy/
This filename has already been used, use a different version. See https://pypi.org/help/#file-name-reuse for more information.
```

Maybe this means having to cut a new release?

**Comment by rgommers** [comment]
We may anyway have to do that, the versions are outdated (both Python and NumPy). I just updated SciPy to `numpy>=1.16.5` for the next release (https://github.com/scipy/scipy/pull/12846). That is 3-4 months away so maybe we have to keep 1.15.x, but Python 3.5 / NumPy 1.13.3 is definitely too old right now.

**Comment by rgommers** [comment]
Also, taking over https://github.com/scipy/scipy/pull/12803 will be useful.

**Comment by astrofrog** [comment]
@rgommers - sounds good, I agree dropping Python 3.5 makes sense. Is there any good reason to change the min versions of Numpy for Python 3.6+?

**Comment by rgommers** [comment]
Good question - on second thought, it's fine as is. It's only for building after all, and the current versions are fine ABI-wise. I was conflating this with runtime dependencies, which are the ones that should be bumped according to NEP 29.

**Comment by astrofrog** [comment]
@rgommers - I agree that we should leave it as-is. I think we also want to keep e.g. Python 3.5 following the note here: https://github.com/scipy/oldest-supported-numpy/commit/f8fb249e2a925018fd6e48037c4ab35bb899d48a

**Comment by astrofrog** [comment]
I released v0.4 which includes the source distribution

**Comment by rgommers** [comment]
Excellent, thanks @astrofrog
